libs/area_reporting.py

```diff
@@ -31,7 +31,8 @@ def __init__(self, config, area):
             camera['last_processed_time'] = time.time()
 
         self.mail_service = MailService(config)
```

Comment:
I know this wasn't edited in this change, but presumably an `if self.should_send_email_notifications` should be here, no? :sweat_smile:

Reply:
```suggestion
        if self.should_send_email_notifications:
            self.mail_service = MailService(config)
```